### PR TITLE
Test for the labeler.

### PR DESCRIPTION
**Pull request by cmarmo**
#### What does this implement/fix? Explain your changes.
Test for the labeler.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -40,6 +40,8 @@
 **scikit-learn** is a Python module for machine learning built on top of
 SciPy and is distributed under the 3-Clause BSD license.
 
+This is a test for the labeler.
```
Comment (cmarmo) [inline README.rst:43]: This is my review

**Review by cmarmo — COMMENTED** [review]
LGTM

**Review by alfaro96 — COMMENTED** [review]
Let us see if the workflow works!

**Comment by alfaro96** [comment]
Hey @cmarmo,

The workflow is not working because

```
curl -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" -d '{"labels": ["Waiting for Reviewer"]}' https://api.github.com/repos/${{ github.repository }}/pull/${{ github.event.pull_request.number }}/labels
```

shoud be

```
curl -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" -d '{"labels": ["Waiting for Reviewer"]}' https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/labels
```

**Comment by alfaro96** [comment]
Hey @cmarmo,

The workflow raises the following "error":

```
{
  "message": "Not Found",
  "documentation_url": "https://docs.github.com/rest"
}
```

An explanation of how to solve the issue is here: https://github.com/scikit-learn/scikit-learn/pull/18500#discussion_r498720560.

When the issue has been solved, I will review this PR again and check if it works then.

Thank you!

**Review by alfaro96 — COMMENTED** [review]
Hey @cmarmo, a "fake" review here :)

**Comment by alfaro96** [comment]
Hum, still failing 😕.

**Comment by cmarmo** [comment]
> Hum, still failing confused.

Yep... thanks for your help anyway... not sure I can reproduce the behavior of the main repo... I will take some more time to figure it out..

File: README.rst [hunk 2]
```diff
@@ -40,6 +40,8 @@
...
+
```
Comment (jnothman) [inline README.rst:44]: Can I review here?

Comment (cmarmo) [reply README.rst:44]: You're welcome! But this still doesn't work... :(
Thanks! :)

File: README.rst (hunk 1 above)
Comment (alfaro96) [reply README.rst:43]: Is now suppose to work?

**Review by alfaro96 — APPROVED** [review]
Let us see if it works now :)

**Review by alfaro96 — COMMENTED** [review]
Hey @cmarmo!

I have seen a new version of the workflow. I am here to check if it works :)

**Comment by alfaro96** [comment]
And it works :)

**Comment by cmarmo** [comment]
> And it works :)

Yes!! :)
I just discovered that pull requests do not have labels in the github rest API only issues...

**Comment by alfaro96** [comment]
> > And it works :)
> 
> Yes!! :)
> I just discovered that pull requests do not have labels in the github rest API only issues...

it is surprising (and I little bit confusing) to treat PRs as issues in the REST API 😕

**Review by cmarmo — COMMENTED** [review]
Check behavior of this version

**Pull request closed without merging**